modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
@rhutchison We don't need this. It's just a duplication of *article.client.service*